centos/7/Dockerfile

```diff
@@ -1,6 +1,8 @@
-FROM centos/aarch64:latest
+FROM centos:7
```

Comment:
Will not this pull an amd64 image?

Reply:
CentOS uses multi-arch images now: https://hub.docker.com/layers/centos/library/centos/7/images/sha256-1f832b4e3b9ddf67fd77831cdfb591ce5e968548a01581672e5f6b32ce1212fe?context=explore

Reply:
Thanks for the clarification!